tcms/templates/case/get_details.html

```diff
@@ -65,23 +65,27 @@ <h4>{% trans "Comments" %}:</h4>
 				<ul class="comment ul-no-format">
 				{% for comment in case_comments %}
 					<li>
-						<span class="strong">#{{ forloop.counter }}</span>
-						<span class="strong">
-                                                    <a href="{% url "tcms-profile" comment.user.username %}">
-                                                       {{ comment.user.username }}
-                                                    </a>
-                                                </span>
-						<span class="grey">{{ comment.submit_date }}</span>
-						<br/>
+                        <div>
+                            <div>
+                                <span class="strong">#{{ forloop.counter }}</span>
+                                <span class="strong">
```

Comment:
Original file uses tabs, not spaces so use tabs as well. This will make the diff show only what has actually been changed. (Tip: configure your editor to not replace tabs with spaces).

Reply:
I fix my editor settings, but note that I moved all the lines a few tabs left, in order to aline them.

Reply:
You will have to undo this b/c now the tags don't align with the for loop on line 66.

Also what about the ul tag on line 65 ? With the new changes it looks like this needs to be removed.